### DISCUSSION TITLE
tdengine性能问题

### DISCUSSION THREAD

**Discussion by dullduck** [taosdata/TDengine#5249] [Q&A]
我在8核8G的机器上测试性能，为何只能达到官方的3分之1，或者4分之1呢？
下面是一个client，一次一条数据。只有5k，官方15k的吞吐量
[root@ tdengine]# ./tdengineTest -dataDir /root/robin/taosdb/testdata/ -numOfFiles 1 -clients 1 -rowsPerRequest 1 -w 1
write data
---- clients: 1
---- dataDir: /root/robin/taosdb/testdata/
---- numOfFiles: 1
---- rowsPerRequest: 1
Thread 1, writing sID 0, eID 0
open file /root/robin/taosdb/testdata//testdata0.csv success
---- Spent 189.524994 seconds to insert 1000000 records, speed: 5276.349121 Rows/Second

下面是一个client，一次100条数据。只有19k，官方65k的吞吐量
[root@tdengine]# ./tdengineTest -dataDir /root/robin/taosdb/testdata/ -numOfFiles 1 -clients 1 -rowsPerRequest 100 -w 1
write data
---- clients: 1
---- dataDir: /root/robin/taosdb/testdata/
---- numOfFiles: 1
---- rowsPerRequest: 100
Thread 1, writing sID 0, eID 0
open file /root/robin/taosdb/testdata//testdata0.csv success
---- Spent 5.198000 seconds to insert 1000000 records, speed: 192381.687500 Rows/Second

**Comment by yu285** [comment]
和机器性能，还有建表方式都有关系。
